papas/detectors/CMS.py

```diff
@@ -1,25 +1,31 @@
-from detector import Detector, DetectorElement
-import material
-from geometry import VolumeCylinder
+from heppy.papas.detectors.detector import Detector, DetectorElement
+import heppy.papas.detectors.material as material
+from heppy.papas.detectors.geometry import VolumeCylinder
 import math
 import random
 
 class ECAL(DetectorElement):
 
     def __init__(self):
         volume = VolumeCylinder('ecal', 1.55, 2.1, 1.30, 2. )
-        mat = material.Material('ECAL', 8.9e-3, 0.275)
+        mat = material.Material('ECAL', 18.062e-3, 0.275)
```

Comment:
radiation length is not supposed to have an effect. 
try tuning interaction length instead